e2e/tests/app.e2e.js

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from '@playwright/test';
+import {NavBarPage} from '../pages/nav-bar-page';
+import {HomePage} from '../pages/home-page';
+
+test.describe('AngularJS-Material-Vite app', () => {
+    test.beforeEach(async ({ page }) => {
+        await page.goto('http://localhost:3000/');
+    });
+    test.describe('TopNav', () => {
+        test('Navbar', async ({ page }) => {
+            const navBarPage = new NavBarPage(page);
+
+            await expect(navBarPage.logo).toHaveAttribute('src', '/assets/img/icon-ng-hotrod-light.svg');
+            await expect(navBarPage.header).toHaveText('ng-hotrod');
+        });
+        test('Redirect to GitHub page', async ({page}) => {
+            const navBarPage = new NavBarPage(page);
+
+            await navBarPage.clickOnGitHubIcon();
+
+            await expect(navBarPage.githubRepoName).toHaveText('angularjs-material-vite');
+        });
+    })
+
+    test.describe('Home', () => {
+        test('HomePage content', async ({ page }) => {
+            const homePage = new HomePage(page);
+
+            await expect(homePage.title).toHaveText('Modern starter for AngularJS + Angular Material');
+            await expect(homePage.heroImage).toHaveAttribute('src', 'https://res.cloudinary.com/fyodorio/image/upload/v1611994660/my-logos/hotrod-logo_rhkfxy.jpg');
+            await expect(await homePage.featuring.all()).toHaveLength(8);
+        });
+        test('Redirect to Angular Material page', async ({page}) => {
+            const homePage = new HomePage(page);
+
+            await homePage.clickOnAngularMaterialLink();
+
+            await expect(page.url()).toContain('angularjs.org');
+        });
+    })
+
+
+});
```

Comment:
Some of the e2e tests are failing for me.
Does everything pass for you?

Reply:
Yes, all tests passed
<img width="246" alt="image" src="https://user-images.githubusercontent.com/5073455/220632870-a2afa452-03df-4321-9d24-0f13598e8a43.png">

Reply:
They pass now for me too 🤷‍♂️